docs/source/api/notebooks/Notebook_6_API2CLI.ipynb

```diff
@@ -0,0 +1,336 @@
+    "\n",
+    "Then we just add two addition CLI launch arguments, and run the CLIs below on the clusters within running docker environment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# CLI for the customized RGAT model training\n",
+    "!python -m graphstorm.run.launch \\\n",
+    "           --part-config ./acm_gs_2p/acm.json \\\n",
+    "           --num-trainers 4 \\\n",
+    "           --num-servers 1 \\\n",
+    "           --num-samplers 0 \\\n",
+    "           --ip-config ./ip_list.txt \\\n",
+    "           --ssh-port 2222 \\\n",
+    "           demo_run_train.py --cf acm_nc.yaml \\\n",
+    "                             --save-model-path models/ \\\n",
+    "                             --node-feat-name paper:feat author:feat subject:feat \\\n",
+    "                             --num-epochs 5 \\\n",
+    "                             --rgat-encoder-type ara\n",
+    "\n",
+    "# CLI for the customized RGAT model inference\n",
+    "!python -m graphstorm.run.launch \\\n",
+    "           --part-config ./acm_gs_2p/acm.json \\\n",
+    "           --num-trainers 4 \\\n",
+    "           --num-servers 1 \\\n",
+    "           --num-samplers 0 \\\n",
+    "           --ip-config ./ip_list.txt \\\n",
+    "           --ssh-port 2222 \\\n",
+    "           demo_run_infer.py --cf acm_nc.yaml \\\n",
+    "                             --restore-model-path models/epoch-4 \\\n",
+    "                             --save-prediction-path predictions/ \\\n",
+    "                             --save-embed-path embeddings/ \\\n",
+    "                             --node-feat-name paper:feat author:feat subject:feat \\\n",
+    "                             --rgat-encoder-type ara"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run CLIs on an Amazon SageMaker Cluster\n",
+    "\n",
+    "In order to run the customized models on an Amazon SageMaker cluster, we need to conduct four additional operations:\n",
+    "\n",
+    "1. Partition the ACM data in multiple partitions, e.g., 2 partition, and upload them to an Amazon S3 location, e.g., `s3://<PATH_TO_DATA>/acm_gs_2p`.\n",
+    "2. Upload the configuration yaml file to an Amazon S3 location, e.g., `s3://<PATH_TO_TRAINING_CONFIG>/acm_nc.yaml`.\n",
+    "3. Git clone [GraphStorm source code](https://github.com/awslabs/graphstorm), and move the `demo_run_train.py` and `demo_run_infer.py` files from the `graphstorm/docs/source/api/notebooks/` folder to the `graphstorm/python/graphstorm/` folder.\n",
+    "4. Follow the [Setup GraphStorm SageMaker Docker Image](https://graphstorm.readthedocs.io/en/latest/cli/model-training-inference/distributed/sagemaker.html#step-1-build-a-sagemaker-compatible-docker-image) tutorial to create a docker image. When building the docker image, please make sure the local graphstorm source code have the two Python files moved.\n",
```

Comment:
What does `the local graphstorm source code have the two Python files moved` mean?

Reply:
Just want to confirm the Step 3 is done.  Removed to avoid confusion.